include/kos/mutex.h

```diff
@@ -211,7 +211,7 @@ int mutex_lock_timed(mutex_t *m, int timeout) __nonnull_all;
     \retval 0               If the mutex is not currently locked
     \retval 1               If the mutex is currently locked
 */
-int mutex_is_locked(const mutex_t *m) __nonnull_all;
+int __pure mutex_is_locked(const mutex_t *m) __nonnull_all;
```

Comment:
It's now sandwitched between two attributes, which looks a bit weird, but not enough for me to call a respin.